lib/generators/components/templates/component_test.rb.erb

```diff
@@ -0,0 +1,11 @@
+require 'component_test_helper'
+
+class <%= @test_name %>Test < ComponentTestCase
```

Comment:
Do we have tests for this in the dummy application? This should be failing I'd have thought?